.github/workflows/linkchecker.yaml

```diff
@@ -0,0 +1,24 @@
+name: Run linkchecker on site
+
+on:
+  schedule:
+    - cron:  '0 0 1,15 * *'
+
+jobs:
+  run-linkchecker:
+    if: github.repository == 'canonical-web-and-design/charmhub.io'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout master
+      - uses: actions/checkout@v2
+
+      - name: Install linkchecker
+        run: sudo pip install LinkChecker
+
+      - name: Run linkchecker
+        run: linkchecker --threads 2 --ignore-url "http://localhost*" --ignore-url "https://assets.ubuntu.com$" --check-extern --no-warnings https://charmhub.io
+
+      - name: Send message on failure
+        if: failure()
+        run: curl -X POST -F "workflow=${GITHUB_WORKFLOW}" -F "repo_name=${GITHUB_REPOSITORY}" -F "action_id=${GITHUB_RUN_ID}" ${{ secrets.BOT_URL }}?room=%23web-team
```

Comment:
not sure about the ping on the webteam channel

Reply:
@anthonydillon is using [docs](https://github.com/canonical-web-and-design/multipass.run/blob/4818bab04a16028ca67cca5367f6817a30fe73ee/.github/workflows/docs-links.yaml#L24) for Docs link check. Shall we create a separate channel for linkchecker notifications?

Reply:
Webteam would be fine as its information we should want to know. Creating a new room is likely for these notifications to be missed,

Reply:
In that case I keep it as it is.

Reply:
with the `--check-extern` if any of the charms publisher posts a 404 links we will always get this message until the publisher fixes it